### ISSUE TITLE
Error in version 1.10.4 - Failed to load the autocomplete-python package and Error compiling Less stylesheet: 

### ISSUE THREAD

**Issue by JonathanMortes** [autocomplete-python/autocomplete-python#351]
**Atom**: 1.22.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [autocomplete-python](https://github.com/autocomplete-python/autocomplete-python) package 1.10.4

### Stack Trace

Failed to load the autocomplete-python package

```
At error evaluating function `rgba`: color functions take numbers as parameters in /Users/jonathanmortesesquer/.atom/packages/autocomplete-python/node_modules/kite-installer/styles/variables.less:8:24

LessError: error evaluating function `rgba`: color functions take numbers as parameters
  at /packages/autocomplete-python/node_modules/kite-installer/styles/variables.less:8:24
```

### Commands

```
```

### Non-Core Packages

```
autocomplete-python 1.10.4 
busy-signal 1.4.3 
django-templates 0.5.0 
intentions 1.1.5 
linter 2.2.0 
linter-flake8 2.2.1 
linter-python-pep8 0.2.0 
linter-ui-default 1.6.10 
```

**Comment by pyvim** [comment]
Looks like fixed in v1.10.5.

**Comment by dezoito** [comment]
> Looks like fixed in v1.10.5.

It is working now, indeed.
